geomstats/geometry/stratified/graph_space.py

```diff
@@ -35,15 +57,54 @@ def _pad_graph_points_with_zeros(points, n_nodes, copy=False):
 
 
 def _pad_array_with_zeros(array, n_nodes):
+    r"""Pad graphs represented as array with zeros.
+
+    Graph space is an embedding for adjacency matrices of the same dimension. Smaller
+    graphs can be padded adding zero nodes and edges, i.e., block of zero rows and
+    columns.
+
+    Parameters
+    ----------
+    array : array-like, shape=[n_obs, n_original_nodes, n_original_nodes]
```

Comment:
I assume n_obs means n observations?

We choose to write shape=[..., n_original_nodes, n_original_nodes] instead, so that we indicate that the first axis can be omitted if there is one observation.